lib/internal/url.js

```diff
@@ -472,8 +472,8 @@ class URLSearchParams {
       throw new ERR_MISSING_ARGS('name', 'value');
     }
 
-    name = StringPrototypeToWellFormed(name);
-    value = StringPrototypeToWellFormed(value);
+    name = StringPrototypeToWellFormed(`${name}`);
+    value = StringPrototypeToWellFormed(`${value}`);
```

Comment:
why this change? The primordial already coerces to string, so this is just redundant.

Reply:
It throws error:

```
❯ out/Release/node test/parallel/test-whatwg-url-custom-searchparams.js
node:internal/url:590
    value = StringPrototypeToWellFormed(value);
            ^

TypeError: String.prototype.toWellFormed called on null or undefined
```

Reply:
ok, but surely the string null or undefined isn’t what’s desired here - that exception seems like it’s exposing a bug?

Reply:
`String.prototype.toWellFormed(null)` works but `StringPrototypeToWellFormed(null)` doesn't. @legendecas any ideas?

Reply:
That’s because the method takes no arguments, and the primordial operates on the first argument as the receiver. If you use .call on the method it will throw the same.

Reply:
looks like browser USP stringifies these too, so this is indeed the correct change, it's just a surprising API design :-/ feel free to resolve this.